skiko/build.gradle.kts

```diff
@@ -221,10 +219,9 @@ val Project.supportAndroid: Boolean
     get() = findProperty("skiko.android.enabled") == "true" // || isInIdea
 
 kotlin {
+    jvmToolchain(11)
```

Comment:
Does this property override our JDK that we set in the Project settings?

If this is enabled, I have java.vendor = Eclipse Adoptium, java.version = 11.0.18, so our test don't run.

Probably worth to write a comment for the JBR test, that we need to comment this property first.

Reply:
We can revert this I think, was more useful when we had bumped the JDK. It shouldn't override anything though, in a Gradle project the IDE project settings are derived from Gradle.